jekyll/_cci2/orb-intro.adoc

```diff
@@ -12,15 +12,25 @@ contentTags:
 
 Use orbs to:
 
-* Simplify configuration (`.circleci/_config.yml`)
-* Automate repeated processes
-* Accelerate project setup
-* Simplify integration with third-party tools
+* Simplify CI/CD configuration.
+* Automate repeated CI/CD processes.
+* Accelerate project setup.
+* Simplify integration with third-party tools.
 
 [#introduction]
 == Introduction
 
-Orbs are reusable packages of parameterizable configuration that can be used in any project. They are made up of reusable configuration elements, for example, xref:reusing-config#authoring-parameterized-jobs[jobs], xref:reusing-config#authoring-reusable-commands[commands], and xref:reusing-config#executor[executors]. Orbs are available for many languages, platforms, services, and tools. Visit the link:https://circleci.com/developer/orbs[Orbs Registry] to search for orbs to help simplify your configuration.
+Orbs are reusable packages of parameterizable configuration that can be used in any project. They are made up of reusable configuration elements, for example:
+* xref:reusing-config#authoring-parameterized-jobs[jobs]
+* xref:reusing-config#authoring-reusable-commands[commands]
+* xref:reusing-config#executor[executors].
+
+CircleCI supports two types of orbs:
+
+*Registry orbs*:: Orbs that are developed and published using the orb publishing process. Semver-based versioning is used to manage the versions of registry orbs. Registry orbs are available from the link:https://circleci.com/developer/orbs[Orbs Registry]
+*URL orbs*:: Orbs that are stored in a git repository and are not published to the Orbs Registry. URL orbs do not follow semantic versioning, and can be accessed directly using an `https` URL. An org-level allow list is used to manage the addresses of URL orbs that can be used in a project in that organization.
```

Comment:
nit: URL orbs don't strictly have to be stored in a git repository. Though we expect it to be the most common case. 
Any publicly available content URL (e.g. a public S3 object) would also work (as long as the URL prefix is in the org's allow list)